drizzlepac/haputils/align_utils.py

```diff
@@ -725,6 +730,87 @@ def find_alignment_sources(self, output=True, dqname='DQ', crclean=False,
             self.imghdu.close()
             self.imghdu = None
 
+class SBCHAPImage(HAPImage):
+    def __init__(self, filename):
+        super().__init__(filename)
+        self.exptime = max(self.imghdu[0].header['exptime'], 1.0)
+
+    def find_alignment_sources(self, output=True, dqname='DQ', crclean=False,
+                               **alignment_pars):
+        """Find sources in all chips for this exposure."""
+        # Only 1 chip, no need to loop
+        chip = 1
+
+        # find sources in image
+        if output:
+            outroot = '{}_sci{}_src'.format(self.rootname, chip)
+        else:
+            outroot = None
+
+        sciarr = self.imghdu[("SCI", chip)].data.copy()
+
+        # Remove all background noise
+        # This background noise is effectively integerized by the SBC detector
+        sci_gauss = ndimage.gaussian_filter(sciarr, sigma=3.0)
+        bkg_noise = astropy.stats.sigma_clipped_stats(sci_gauss, maxiters=1)
+        # bkg_noise = astropy.stats.sigma_clipped_stats(sciarr[sciarr > 0], maxiters=1)
```

Comment:
Since bkg_noise is really three variables, I would add a comment to make this clear (e.g., mean, median, stddev) at Line 755.

Reply:
Fixed in 7b869e6.